src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/SearchByEmailController.cs

```diff
@@ -0,0 +1,94 @@
+﻿using FluentValidation;
+using FluentValidation.AspNetCore;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Provider.PR.Domain.Interfaces;
+using SFA.DAS.Provider.PR.Domain.OuterApi.Responses;
+using SFA.DAS.Provider.PR.Web.Authorization;
+using SFA.DAS.Provider.PR.Web.Infrastructure;
+using SFA.DAS.Provider.PR.Web.Infrastructure.Services;
+using SFA.DAS.Provider.PR.Web.Models.AddEmployer;
+using SFA.DAS.Provider.PR.Web.Models.Session;
+
+namespace SFA.DAS.Provider.PR.Web.Controllers.AddEmployer;
+
+[Authorize(Policy = nameof(PolicyNames.HasContributorOrAbovePermission))]
+
+[Route("/{ukprn}/addEmployer/searchByEmail", Name = RouteNames.AddEmployerSearchByEmail)]
+public class SearchByEmailController(IOuterApiClient _outerApiClient, ISessionService _sessionService, IValidator<SearchByEmailSubmitViewModel> _validator) : Controller
+{
+    public const string ViewPath = "~/Views/AddEmployer/SearchByEmail.cshtml";
+    public const string MultipleAccountsShutterPathViewPath = "~/Views/AddEmployer/MultipleAccountsShutterPage.cshtml";
+
+    [HttpGet]
+    public IActionResult Index([FromRoute] int ukprn)
+    {
+
+        var viewModel = GetViewModel(ukprn);
```

Comment:
I do not see email being retried from session here?

Reply:
oh sorry I confused this with clearing the sessionModel. I've implemented it as a reload, and added a cleardown in the StartController